test/test_metalearning/pyMetaLearn/test_meta_features.py

```diff
@@ -161,199 +168,279 @@ def meta_train_data_transformed(request):
 
 
 def test_number_of_instance(meta_train_data):
-    X, y, categorical = meta_train_data
+    X, y, feat_type = meta_train_data
+    categorical_ = {
```

Comment:
Hey, do you think it would be possible to have this translation in the actual meta-feature computation? Then we could skip counting string features.

Reply:
feature_type.py
```python 
sklearn_transf_spec = [
                (name, transformer, feature_columns)
                for name, transformer, feature_columns in [
                    ("text_transformer", self.txt_ppl, text_features),
                    ("categorical_transformer", self.categ_ppl, categorical_features),
                    ("numerical_transformer", self.numer_ppl, numerical_features),
                ]
                if len(feature_columns) > 0
            ]
''''
I just assumed it because it was priviously done with numerical features too.

```python
self.column_transformer = sklearn.compose.ColumnTransformer(
            transformers=sklearn_transf_spec,
            sparse_threshold=float(self.sparse_),
        )
```

But i think this is the part. So the column_transformer gets the `sklearn_transf_spec` and  i think that the order defined apove defines the order the values appaer in X_transformed. This means that i have to change the position of text_transformer.

Reply:
`possible to have this translation in the actual meta-feature computation?` do not really understand what this means. Can you explain this a bit more ?

Reply:
I meant that we pass `feat_type` to the meta-feature computation functions. Then, in the meta-feature computation function we do computations such as:
```
categorical_ = {
    col: True if feat_type.lower() == "categorical" else False
    for col, feat_type in feat_type.items()
}
```
Then, this can be customized to what is required by the metafeature.